sci-libs/mc/mc-1.5-r1.ebuild

```diff
@@ -38,6 +38,7 @@ PATCHES=(
 	"${FILESDIR}"/1.4-multilib.patch
 	"${FILESDIR}"/1.4-doc.patch
 	"${FILESDIR}"/${P}-unbundle.patch
+	"${FILESDIR}"/${P}-missing-decl.patch
```

Comment:
Has this patch been sent upstream?

Reply:
Just sent it to author, Michael Holst.

Reply:
My email delivery failed, it may be bad address in package site.
Error 550: 5.7.1 Relaying denied.

Reply:
According to the changelog, the latest release is from 2010, probably this is a dead project.